### DISCUSSION TITLE
Omit image from top of blog post

### DISCUSSION THREAD

**Discussion by genebean** [thuliteio/doks#883] [Q&A]
Is there a way to allow some blog posts to not have images at the top?

**Comment by h-enk** [comment]
Not now, but will be considering this. Just like w/ making the previous blog post layouts an option.

**Reply by genebean** [reply]
Thank you! Also, I appreciate your engagement here :)

**Reply by mhite** [reply]
@h-enk - curious if this functionality was ever added? thanks!

**Reply by h-enk** [reply]
@mhite, yes sure! Here's how it works:

- If you'd want to show an image to a blog post, add an image with `cover`, `thumbnail`, or `featured` in it's filename (as a page resource),
- If you _don't_ want to show an image to a blog post, do _not_ add an image with `cover`, `thumbnail`, or `featured` in it's filename (as a page resource),
- To disable _all_ images on the blog (list) page, in `config/_default/params.toml` set `doks.imageList = false` (line 70)
- To disable _all_ images for blog posts, in `config/_default/params.toml` set `doks.imageSingle = false` (line 71)

I will add something like this to the docs (I'm currently updating them)

**Reply by mhite** [reply]
Thanks!
